scaffold/template/custom/README.md

```diff
@@ -1,9 +1,62 @@
-# HelloWorld
-
-## Getting started
+# {{cookiecutter.project_slug}}
```

Comment:
```suggestion
# ProjectName
```

This should use the standard rn template placeholders, and our config.js script will pick it up too. We had "HelloWorld" which is now outdated.

Reply:
The reason why, is that the template on its own is always a valid `react-native init --template` valid template, if it doesn't have cookiecutter tags.

Reply:
That's why `cookie.js` was created